source/hassio/enable_i2c.markdown

```diff
@@ -44,3 +44,23 @@ and make sure the first partition is available.
   present at boot time.
 
 The I2C devices should now be present under /dev.
+
+## From Home Assistant Operating System Terminal
+
+
+Alternatively, by attaching a keyboard and screen to your device, you can access the physical terminal to the Home Assistant operating system.
+
+You can enabled i2c via this terminal:
+
+- Login as `root`.
+- Type `login` and press enter to access the shell.
+- Type the following to enable i2c:
+  ```shell
+  mount /dev/sda1 /mnt
+  mkdir -p /mnt/CONFIG/modules
```

Comment:
This seems off, the mount is made on `/mnt`, next the modules folder is created in `/mnt/CONFIG`.

Additionally, is `/dev/sda1` guaranteed/fixed?

Reply:
Unresolved the conversation, as you just confirmed here that this is not stable.

Users will copy and paste this and thus it might not work.

Reply:
This was resolved via mention of other block devices   https://github.com/home-assistant/home-assistant.io/pull/16073/commits/68762a39dea58fb259265caff6cf9c86bf3faab3